extension/manage/js/index.js

```diff
@@ -169,4 +233,32 @@ const STATS_MAP = {
         selector: ".topCratesData",
         color: CHART_COLOR,
     });
-})();
\ No newline at end of file
+}
+
+function yearList() {
+    const y = new Date().getFullYear()
+    const year = document.querySelector(".filter-list")
+    year.addEventListener('click', function (e) {
+        if (e.target.tagName === "LI") {
+            year.childNodes.forEach(i => i.classList.remove("selected"))
+            e.target.className = "selected"
+            render(e.target.innerHTML)
+        }
+    })
+
+    for (let i = y; i > y - 3; i--) {
+        const li = document.createElement('li')
+        li.innerHTML = i
+        if (i === y) {
+            li.className = "selected"
+        }
+        year.append(li)
+    }
+}
+
+(async () => {
+    await render(undefined)
```

Comment:
Calculate the current year as the default year, rather than use the `undefined`.